src/esp/gfx/replay/Recorder.h

```diff
@@ -66,6 +66,11 @@ class Recorder {
    */
   void saveKeyframe();
 
+  /**
+   * @brief Returns the last saved keyframe.
+   */
+  Keyframe getLatestKeyframe();
```

Comment:
How about a const reference instead? This is a large object with several child containers, so a copy is expensive. I guess the python binding will eventually have to make a deep copy (that can't be helped), but we shouldn't encourage copying here in the C++ api.